application/ApplicationRequest.js

```diff
@@ -142,17 +144,15 @@ function getUrl(request, host, data) { // _data = nullable
         return host + '/api/application/servers/' + data +'/unsuspend';
     } else if ('DeleteServer') {
         return host + '/api/application/servers/' + data;
+    } else if('GetServerInfo') {
+        return host + '/api/application/servers/' + data;
     }
 }
 
-function createError(request, err, data) {
+function createError(request, err) {
     let error;
     if (request == 'CreateUser' || request == 'EditUser' || request == 'GetUserInfo') {
-        if(/^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(data.email) == false) {
-            error = new Error('The provided email is not a valid.');
```

Comment:
Why did you remove the email test?

Reply:
So, the version in the NPM don't have this and i'm new on github so i pick up all the file and put here sorry if i make some mistake..

Reply:
i think the version of the github and the NPM are different

Reply:
That isn't a reason why it was removed though?

Reply:
I am just now getting to this, and taking each parts of these step by step. For this part yes the NPM and github parts are different, github version will always be more updated than npm, but the github version is more unstable, therefore reccomended to do development on the github version instead of the NPM files :)